managed-ledger/src/test/java/org/apache/zookeeper/MockZooKeeper.java

```diff
@@ -93,6 +94,7 @@ private void init(ExecutorService executor) {
         } else {
             this.executor = Executors.newFixedThreadPool(1, new DefaultThreadFactory("mock-zookeeper"));
         }
+        this.callbackExecutor = Executors.newFixedThreadPool(4, new DefaultThreadFactory("mock-zookeeper-callback"));
```

Comment:
I kept it more than 1 in case in future if callback which will be executed by this thread again calls mockZk then Single-thread-pool won't be able to proceed further.